Sources/TuistCache/Cache/Resources/CloudCacheResourceFactory.swift

```diff
@@ -23,21 +23,22 @@ class CloudCacheResourceFactory: CloudCacheResourceFactorying {
     }
 
     func existsResource(name: String, hash: String) throws -> CloudExistsResource {
-        let url = try apiCacheURL(name: name, hash: hash, cacheURL: cloudConfig.url, projectId: cloudConfig.projectId)
-        var request = URLRequest(url: url)
-        request.httpMethod = "HEAD"
-        return HTTPResource(
-            request: { request },
-            parse: { _, _ in CloudResponse(status: "HEAD", data: CloudEmptyResponse()) },
-            parseError: { _, _ in CloudEmptyResponseError() }
+        let url = try apiCacheURL(
+            name: name,
+            hash: hash,
+            cacheURL: cloudConfig.url,
+            path: "/api/cache/check_cache_artifact",
```

Comment:
What about a simpler and more explicit /api/cache/exists ?

Reply:
I like this suggestion, updated